src/main/java/org/owasp/validator/html/scan/AntiSamyDOMScanner.java

```diff
@@ -555,7 +553,7 @@ private boolean processAttributes(Element ele, String tagName, Tag tag, int curr
                             promoteChildren(ele);
                             addError(ErrorMessageUtil.ERROR_ATTRIBUTE_CAUSE_FILTER,
                               new Object[]{tagName, HTMLEntityEncoder.htmlEntityEncode(name), HTMLEntityEncoder.htmlEntityEncode(value)});
-
+                            return true;
```

Comment:
@spassarop - Can you review adding this return true here? It was added by @LiuXing-R ~4 months ago and merged into main. I'm guessing you reviewed/OK'd this change? I just want to make sure.  Same for return true added on line 567 below.

Reply:
It's ok.